pkg/controller/nodenetworkconfigurationpolicy/enactmentstatus/conditions/counter.go

```diff
@@ -34,7 +40,51 @@ func (c CountByConditionType) Progressing() int {
 func (c CountByConditionType) Available() int {
 	return c[nmstatev1alpha1.NodeNetworkConfigurationEnactmentConditionAvailable]
 }
+func (c CountByConditionType) Matching() int {
+	return c[nmstatev1alpha1.NodeNetworkConfigurationEnactmentConditionMatching]
+}
+
+func (c ConditionCount) True() CountByConditionType {
```

Comment:
Please make sure not to set functions that are not used externally as public. I'd like to limit access to this module as much as possible. It is quite complex.

Reply:
I want to have it symetrical.